pytest_fixtures/component/leapp_client.py

```diff
@@ -0,0 +1,204 @@
+from broker import Broker
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import PRDS
+from robottelo.hosts import ContentHost
+from robottelo.logging import logger
+
+synced_repos = pytest.StashKey[dict]
+
+RHEL7_VER = '7.9'
+RHEL8_VER = '8.10'
+RHEL9_VER = '9.4'
```

Comment:
Use from constants either or define it there.

Reply:
We don't use anywhere else so I kept these locally.